### PR TITLE
refresh button

### PR DESCRIPTION
**Pull request by anderoo**
👋 Hi there. This PR adds a refresh button to the ArgoCD plugin.

## Screenshots

Screenshot 2021-03-29 at 14 07 43

## Issues

No actual issues however it might be worth considering using the Table `isLoading` rather than the linear loading bar - This avoids resizing on refresh. Happy to add this now if desired, this would look like:

Screenshot 2021-03-29 at 14 11 37

### CODE REVIEW

**Review by dtuite — DISMISSED** [review]
@anderoo looks great but needs a rebase. Thank you!

**Review by Xantier — APPROVED** [review]

File: src/plugin.test.tsx [hunk 1]
```diff
@@ -116,6 +116,34 @@ describe('argo-cd', () => {
...
+
+      const nextResponseStub = getResponseStub;
+      nextResponseStub.status.sync.status = "OutOfSync";
+
+      worker.use(
+        rest.get('*', (_, res, ctx) => res(ctx.json(getResponseStub)))
+      );
+
+      const refreshButton = await rendered.findByTitle("Refresh");
+      refreshButton.click();
```
Comment (Xantier) [inline src/plugin.test.tsx:141]: Something like 
```
fireEvent.click(refreshButton)
```
here, where fireEvent is from `testing-library/react`

**Comment by anderoo** [comment]
😓 Not sure one why tsc is complaining here... `Property 'click' does not exist on type 'HTMLElement'`

File: src/plugin.test.tsx (hunk 1 above)
Comment (anderoo) [reply src/plugin.test.tsx:141]: Thanks

**Review by Xantier — APPROVED** [review]
LGTM, merging.

**Comment by anderoo** [comment]
> LGTM, merging.

Thanks. LMK if the other loading state is desired and happy to PR it.

**Pull request merged**